lib/middleware.js

```diff
@@ -65,6 +65,18 @@ module.exports = function wrapper(context) {
         } catch (e) {
           notFound = true;
         }
+        const currentRequest = Object.assign({}, {
+          url: req.url,
+          method: req.method,
+          headers: req.headers,
+          filename,
+          notFound
+        });
+        if (context.options && context.options.content && context.options.content.filter) {
+          if (!context.options.content.filter(currentRequest)) {
+            return goNext();
+          }
+        }
         if (notFound) {
           return resolve(goNext());
```

Comment:
Question: why is it that every other "goNext" is a regular return but this one is a return resolve(goNext())?

Reply:
Good question but I don't know the answer, try to remove the `resolve` and see what's happening (if something happens). Please add a comment that this is something to investigate and clearify :)